config/default/metrics_service.yaml

```diff
@@ -2,8 +2,11 @@ apiVersion: v1
 kind: Service
 metadata:
   labels:
+    # deprecated, could not be removed because of deployment match selector immutability
     control-plane: vm-operator
-    app.kubernetes.io/name: vm-operator
+
+    app.kubernetes.io/instance: vmoperator
```

Comment:
`app.kubernetes.io/instance` - A unique name identifying the instance of an application
This labels is not relevant at all

Reply:
Why helm sets this label then ?

Reply:
in helm value of this label is taken from release name, it means that you can deploy multiple instances of the same chart with different configurations, etc.

Reply:
The logic could be applied to the operator installed via a tagged manifest. If you just download it and apply it would be default release.  Then you can run it via kustomize (change the instance label, + other stuff)  and deploy a new  operator release.

This way, the operators installed via manifest have a release label, and are also consistent with the operator installed via helm. 

I am not sure about the release name `vmoperator`. Maybe `default` is a better one. I am not sure. WDYT?

Reply:
it's not helm specific, but meaningless in context of plain manifests. set it to default

Reply:
@AndrewChubatiuk fixed, please review